config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -361,3 +361,45 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=40m
       - --use-logexporter
+
+# cluster volume load scaling by number of nodes
```

Comment:
nit: it might be better for the readability to use the similar naming approach with previous storage job definitions, if we can think the PVC like a different volume type.

Reply:
I cant think of a goof name for this test - but this is a different test. I'll put more thought into this, but open to suggestions.